WordPress/gutenberg
# Update template preview dimensions in table layout

Fixes a regression that snuck in somewhere along the way.

## What?
Reduce the size of the template preview in table layout.

## Why?
It is currently much too large.

## How?
* Add a `width` property to `page-templates-preview-field.is-viewtype-table`
  * Also adds a `max-height` to avoid rows growing too large.
* Adjust the column width so that it hugs its contents.

## Testing Instructions
1. Navigate to the templates data view
2. Enable the preview field
3. Observe the updated dimensions

| Before | After |
| --- | --- |
| <img width="586" alt="Screenshot 2024-07-25 at 14 05 54" src="https://github.com/user-attachments/assets/a923debc-3860-423d-82e3-e08f9f48da7a"> | <img width="457" alt="Screenshot 2024-07-25 at 14 04 15" src="https://github.com/user-attachments/assets/ea8e1fd2-ae29-4cb6-9f15-90f65edb5021"> |
